WORKFLOW.rst

```diff
@@ -0,0 +1,216 @@
+====================================
+ RPC OpenStack Development Workflow
+====================================
+
+Summary
+=======
+
+It is currently necessary, to change the way the RPC OpenStack team develops
+the project (i.e., adds features, fixes bugs). The old workflow is described
+in the :ref:`Old Workflow <old-workflow>` section.
+
+The new workflow is outlined under :ref:`Proposed Workflow
+<proposed-workflow>`
+
+.. _old-workflow:
+
+RPC OpenStack's Old Workflow
+============================
+
+Facts
+-----
+
+#. |rpcos| is configured with the following branches:
+
+   - ``master``
+
+   - ``kilo``
+
+   When we decide to support liberty, a third branch will appear - ``liberty``.
+
+#. Bug fixes move from ``master`` to the other branch(es) via cherry-picks.
+
+#. Bugs are hand-managed by developers and backports are tracked haphazardly.
```

Comment:
All bugs that are addressed by changes have to be hand-closed by someone. There is no automation.

"haphazardly" is accurate and factual. We currently rely on people noticing that all backports have been merged before anything is closed. This (I believe) is skewing our Kanban metrics against us and is far more work than we should be doing. GitHub has tooling for closing issues for us, we should be utilizing that, not working against it.